README.md

```diff
@@ -104,6 +106,8 @@ empty commit to the PR*.
 | `context`      | No       | `unit-test`             | A context to use for the status. (Prefixed with `concourse-ci`, defaults to `concourse-ci/status`). |
 | `comment`      | No       | `hello world!`          | A comment to add to the pull request.                                                               |
 | `comment_file` | No       | `my-output/comment.txt` | Path to file containing a comment to add to the pull request (e.g. output of `terraform plan`).     |
+| `target_url`   | No       | `ATC DEFAULT URL`       | The base URL for the Concourse deployment, used for linking to builds.                              |
+| `description`  | No       | `Concourse CI build %s` | The description status on the specified pull request.                                               |
```

Comment:
The example should not include `%s` here anymore since we are no longer doing formatting on it :)

Reply:
I removed `%s` from example, but we still doing the formatting on: https://github.com/adriagalin/github-pr-resource/blob/ag-external-url/github.go#L301